trieste/utils/pareto.py

```diff
@@ -148,3 +148,42 @@ def hypervolume_indicator(self, reference: TensorType) -> TensorType:
         hypervolume = tf.reduce_sum(tf.reduce_prod(upper - lower, 0))
 
         return tf.reduce_prod(reference[None] - min_pfront) - hypervolume
+
+    def get_hypercell_bounds(self, anti_reference: TensorType, reference: TensorType) -> Tuple:
```

Comment:
```suggestion
    def hypercell_bounds(self, anti_reference: TensorType, reference: TensorType) -> Tuple:
```
partly cos we haven't used `get` anywhere else, and also cos I've never really understood the value of `get_xyz`: I mean it's a function so the `get_`-ness is captured in the signature? Dunno, anyway there's a suggestion